src/_includes/components/menu.njk

```diff
@@ -0,0 +1,11 @@
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
```

Comment:
**TIP!**
If you find yourself using more and more SVG's it might be best to check your design to see how many different SVG's you need for the complete site and if this really is the best method to implement them. Because it can get messy rather quickly.

Reply:
Aside of the logo I'm using 3 icons in the website (2 chevrons and 1 hamburger menu) For the chevrons the only solution that I found is to use them inline (because the color needs to change on hover etc) and I think I will do a component for the menu. TBC in the next pull request 🔜

Reply:
Just an idea: 
In that case it might be more interesting to group the icons in a folder apart from components, in order to keep a better oversight of "actual" the components. Something like "includes/icon-components" or maybe "icon-snippets"